ui/.nvmrc

```diff
@@ -1 +1 @@
-v22.7.0
+22.9.0
```

Comment:
It's fine, `v` prefix not mandatory.
- https://github.com/nvm-sh/nvm/issues/3265#issuecomment-1894759452
- https://docs.renovatebot.com/modules/manager/nvm/